src/platform-implementation-js/dom-driver/gmail/views/gmail-route-view/gmail-route-view.ts

```diff
@@ -394,7 +394,7 @@ class GmailRouteView {
       });
       if (isStreakAppId(this._driver.getAppId())) {
         this._driver.getLogger().error(selectorError, {
-          html: censorHTMLstring(previewPaneContainer.innerHTML),
+          html: extractDocumentHtmlAndCss(),
```

Comment:
this is going to log a lot more data to our servers - do you know the rough frequency of these errors? Just want to make sure we don't DDOS ourselves.

Reply:
if its high frequency, we can sample

Reply:
@Macil pointed out the the rate limit implemented for errors https://github.com/InboxSDK/InboxSDK/blob/d2ea8b878c95570204e2d2b9db2c95d89e379c1e/src/common/log-error.ts#L197